android/sdk/src/main/java/org/jitsi/meet/sdk/RNConnectionService.java

```diff
@@ -35,6 +35,8 @@ class RNConnectionService extends ReactContextBaseJavaModule {
 
     private static final String TAG = ConnectionService.TAG;
 
+    public static ReactApplicationContext reactContext;
```

Comment:
Rather than adding a field for the context, let's turn this class into a singleton and add a getIstance inetad please, I think it will simplify things because then we could put the emitEvent method in it too.

The other file can then do `RNConnectionService.getInstance().emitEvent(...)`